### PR TITLE
sssd nightly

### PR DESCRIPTION
**Pull request by ikerexxe**

### CODE REVIEW

File: ipatests/prci_definitions/nightly_latest_sssd.yaml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_replica_promotion.py::TestWrongClientDomain
+        template: *ci-master-latest
+        timeout: 7200
+        topology: *master_1repl
+
+  sssd-fedora/test_smb:
+    requires: [fedora-fedora/build]
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_latest_sssd.yaml:170]: This should be:
    requires: [sssd-fedora/build]
and the definition also needs:
         update_packages: True
         copr: '@sssd/nightly'

Comment (ikerexxe) [reply ipatests/prci_definitions/nightly_latest_sssd.yaml]: Fixed. Thanks for the feedback.

**Comment by flo-renaud** [comment]
@ikerexxe the failure in test_smb is a known issue on IPA side (ticket #[9124](https://pagure.io/freeipa/issue/9124)) that you can ignore for this PR. Feel free to close this PR and push the same changes to https://github.com/freeipa/freeipa/pull/6348.

**Comment by ikerexxe** [comment]
> Feel free to close this PR and push the same changes to [freeipa#6348](https://github.com/freeipa/freeipa/pull/6348).

That was already done so I proceed to close this PR.

**Pull request closed without merging**